R/sf_helpers.R

```diff
@@ -37,9 +37,9 @@ sf_types <- c(GEOMETRY           = "other",
 
 sf_defaults <- function() {
 
-  point_aes          <- GeomPoint$default_aes
-  textpath_aes       <- GeomTextpath$default_aes
-  polygon_aes        <- GeomPolygon$default_aes
+  point_aes          <- get_geom_defaults(GeomPoint)
+  textpath_aes       <- get_geom_defaults(GeomTextpath)
+  polygon_aes        <- get_geom_defaults(GeomPolygon)
```

Comment:
This is also to deal with the unevaluated expressions and the reason we'd require ggplot2 3.5.2